api/client/run.go

```diff
@@ -254,6 +255,16 @@ func (cli *DockerCli) CmdRun(args ...string) error {
 
 	// Attached mode
 	if *flAutoRemove {
+		// Warn user if they detached us
+		js, err := cli.client.ContainerInspect(createResponse.ID)
+		if err != nil {
+			return runStartContainerErr(err)
+		}
+		if js.State.Running == true || js.State.Paused == true {
+			fmt.Printf("\ndocker: detached from %s! Waiting for it to terminate in order to uphold \"--rm\".\n",
```

Comment:
almost there @mlaventure!

You need to use `fmt.Fprintf` https://golang.org/src/fmt/print.go?s=4905:4982#L176

Hint: The cli has references to the configured error and output streams. See usages in this same file.